company/overview.md

```diff
@@ -0,0 +1,52 @@
+---
+description: 'Company, connections and accounts API overview'
+---
+
+# Company, connections and accounts
+
+## Tradecloud supply chain network
+
+The Tradecloud network consists of many buyers and supplier companies you can connect with.
+
+## Your company
+
+Your company has a profile, containing your company name, logo, info and settings.
+
+[Get profile and settings](get.md)
+
+
+[Update profile and settings](update.md)
+
+TO DO: Upload company logo
+
+## Your connections
+
+When you start on Tradecloud you will have to connect to your existing relations. Also you can find and connect to new relations.
+
+You can list your existing connections using:
+
+[List your connected companies](list-connected-companies.md)
+
+To connect first search for a user of an existing or new relation:
+
+[Search for new companies](search-new-companies.md)
+
+When you found a user and company to connect with, you can request a connection:
+
+[Connect to a new company](connect-to-company.md)
+
+## Your accounts
+
+Each connection has an account code. This is the debitor/creditor/relation code in your ERP system.
+
+The account code is needed when you want to trade with your connection using an ERP integration, either API or connector. Tradecloud uses this account code to route the order, shipment, forecast to the connection.
+
+TO DO: Add account code
```

Comment:
To be resolved before merge.

Reply:
Added to TC-5385 Create API v2 manual for buyer order process (part 2)